flink-flux-core/src/main/java/com/uber/athena/flux/flink/compiler/impl/datastream/FluxContext.java

```diff
@@ -52,7 +53,7 @@
   /**
```

Comment:
All of the components here in the map should be extending from `CompilerVertex` so that the context does not depend on any specific Compiler impl.